crates/sui-core/src/authority_active/checkpoint_driver/mod.rs

```diff
@@ -0,0 +1,878 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{
+    collections::{BTreeMap, BTreeSet},
+    sync::Arc,
+    time::Duration,
```

Comment:
I may be wrong, but I think that you need to import `tokio::time::Duration` rather than `std::time::Duration` if you want to use the simulated-time tests integrated with tokio?